docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -32,34 +32,81 @@ environment for debugging. The tool is meant to be used for debugging a single
 Tetragon agent node. Note that in the context of Kubernetes, the command needs
 to be run from inside the Tetragon Pod's container.
 
+**Key Information Collected by the Bugtool:**
+
+- Tetragon configuration
+- Network configuration
+- Kernel configuration
+- eBPF maps
+- Process traces (if tracing is enabled)
+
+## Running tetra bugtool Demonstrations
+
+### Kubernetes
+
+1. **Identify Tetragon Pod:**
+
+```bash
+kubectl get pods -n <tetragron-namespace> -l app.kubernetes.io/name=tetragon
+```
```

Comment:
You can shift by two your codeblocks so that they fit into the item indentation.
```suggestion
  ```bash
  kubectl get pods -n <tetragron-namespace> -l app.kubernetes.io/name=tetragon
  ```
```

Reply:
shifted by three whitespace to align with indentation